packages/super-components/components/msc-tail-logs/vue/.npmignore

```diff
@@ -0,0 +1,2 @@
+!dist/
+src/
```

Comment:
```suggestion
```

Reply:
we don't need to publish src in the npm component